src/lib/edje/Edje_Common.h

```diff
@@ -1894,7 +1894,11 @@ typedef enum _Edje_Tween_Mode
    EDJE_TWEEN_MODE_CUBIC_BEZIER      = 11,       /**< Cubic Bezier tween mode value */
    EDJE_TWEEN_MODE_LAST              = 12,       /**< Last tween mode value */
    EDJE_TWEEN_MODE_MASK              = 0xff,     /**< Mask tween mode value */
+#ifdef _MSC_VER
+   EDJE_TWEEN_MODE_OPT_FROM_CURRENT  = INT_MAX /**< Options from current tween mode value */
```

Comment:
I didn't get why this is the way to solve it, could you explain to me? some ELI5 haha

Reply:
An enumeration has a size (at least enough to represent an integer) and so they cannot represent values out of its range. For example if in some architecture the integer holds 32 bits, so its maximum value will be 2^31-1, and `(1<<31)` is `2^32`.